packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -418,22 +418,6 @@ describe('Collection aggregations tab', function () {
 
     await waitForAnyText(browser, await browser.$(Selectors.stageContent(1)));
 
-    // make sure it complains that it must be the last stage
```

Comment:
We don't run pipeline anymore for this case, validation is a bit stricter in the new pipeline builder